libpam/README.md

```diff
@@ -140,3 +143,11 @@ timebased, use the `google-authenticator` binary to generate a secret key in
 your home directory with the proper option.  In this mode, clock skew is
 irrelevant and the window size option now applies to how many codes beyond the
 current one that would be accepted, to reduce synchronization problems.
+
+## allow_perm
+
+By default, the PAM module requires the secrets file to be readable only by the owner of the file ( 0400 by default). In situations where the module is not used in a non-default configuration, an administrator may need more leanient file permissions, or a specific setting for their use case.
```

Comment:
For the use-case at my work, the permissions ended up looking like:
`-r--r----- pantsman0 radiusd pantsman0.secret (0440 permissions)`
The radius daemon ran as radiusd and the options provided to the pam module looked like:
`user=radiusd secret=/var/spool/google/${USER}.secret allow_perm=440 no_strict_owner`

This was selected as we wanted as few services running as root as possible, and centralized storage of the secrets for replication to a DR site; unfortunately, that imposed extra requirements for the secret file permissions that were not an issue when the file synchronization daemon and the radius daemon were running as root.

Despite all users having rwx access to the directory, the sticky-bit setting prevents them from deleting the files that belong to the other users, and they can only delete/update their own secrets file.